src/data/FAQdata.ts

```diff
@@ -31,7 +31,7 @@ const data = {
         "(1) the court does not believe that vacating your conviction will be in the interest of society, ",
       body3: "(2) you violated probation or have not paid off your fines, ",
       body4:
-        "(3) you are otherwise not eligible to vacate that particular conviction or ",
+        "(3) you are otherwise not eligible to vacate that particular conviction, or ",
```

Comment:
I can't leave a comment on it, but on line 29 there is a missing space between `judge.The` in the body1 text

Reply:
good catch!